### ISSUE TITLE
[FR DatetimeV2] No consistency between hour resolution order

### ISSUE THREAD

**Issue by RomainWilbert** [Microsoft/Recognizers-Text#500]
"12h30" => 2 resolutions: first is Timex T00, second is Timex T12
"12:30" => 2 resolutions: first is Timex T12:30, second is Timex T00:30

This makes it hard to parse result when implementing rules based on resolution order (minest first, highest last)

**Comment by tellarin** [comment]
The HHhMM hour format was not previously supported.

PR https://github.com/Microsoft/Recognizers-Text/pull/498 has implemented it 5h ago. It will be available on the next package release.

I'll close this issue, OK?
